BookStore/controllers/user.js

```diff
@@ -32,4 +32,27 @@ exports.update = (req, res) => {
             req.user.salt = undefined;
             res.json(user);
         })
-}
\ No newline at end of file
+}
+
+exports.addOrderToUserHistory = (req, res, next) => {
+    let history = [];
```

Comment:
`history` can be a `const` here.

Even if you `.push()` elements inside an array, it can still be a `const`.

Reply:
ok